apps/web/src/components/v2/thread-view.tsx

```diff
@@ -53,19 +56,34 @@ export function ThreadView({
     useState<ManagerGraphState["plannerSession"]>();
   const [programmerSession, setProgrammerSession] =
     useState<ManagerGraphState["programmerSession"]>();
+  const [isTaskSidebarOpen, setIsTaskSidebarOpen] = useState(false);
+  const [plannerTaskPlan, setPlannerTaskPlan] = useState<TaskPlan>();
+  const [programmerTaskPlan, setProgrammerTaskPlan] = useState<TaskPlan>();
```

Comment:
only need to track it in one place
```suggestion
  const [taskPlan, setTaskPlan] = useState<TaskPlan>();
```